function-aws-test/src/test/java/io/micronaut/function/aws/test/EagerSingleton.java

```diff
@@ -4,4 +4,8 @@
 
 @Singleton
 public class EagerSingleton {
+
```

Comment:
Can we use a different bean? I guess `EagerSingleton` is used in a different test. I would like to keep tests separate.

Reply:
Moreover, should we have the been return something else like `return "goodby %s".formatted(name);` to verify a mock bean is actually used.